### ISSUE TITLE
Tweak banner spacing

### ISSUE THREAD

**Issue by ericrwolfe** [mapbox/mapbox-navigation-ios#792]
@frederoni despite the fixes in #790, it seems the spacing and multi-line wrapping is still slightly off, at least when swiping the banners:

Route: Downtown SF to SFO.

![img_0001](https://user-images.githubusercontent.com/664805/32399754-0d881a40-c0b6-11e7-94d2-a8e018770b12.PNG)
![img_0002](https://user-images.githubusercontent.com/664805/32399755-0dbf25c6-c0b6-11e7-910e-aeec64ee5330.PNG)
![img_0003](https://user-images.githubusercontent.com/664805/32399757-0dd9af7c-c0b6-11e7-908e-42de525c75cd.PNG)

**Comment by bsudekum** [comment]
Is this still an issue?

**Comment by bsudekum** [comment]
@ericrwolfe is this still an issue?

**Comment by ericrwolfe** [comment]
@bsudekum @frederoni Yup, same issues as above still exist in the current version of the Nav SDK. (See Exit 423C on the way to SFO from downtown SF)

Most importantly, the secondary label doesn't seem to wrap text.
